### PR TITLE
Create PosAng, OpticsCorr, PSDiag and CurrInfo subpackages

### PR DESCRIPTION
**Pull request by anacso17**

### CODE REVIEW

File: siriuspy/siriuspy/currinfo/lifetime/main.py [hunk 1]
```diff
@@ -0,0 +1,166 @@
+"""Main Module of the IOC Logic."""
+
+import warnings
+import time as _time
+from collections import deque as _deque
+import numpy as _np
+from epics import PV as _PV
+
+from siriuspy.callbacks import Callback as _Callback
```
Comment (xresende) [inline siriuspy/siriuspy/currinfo/lifetime/main.py]: @anacso17 , we are trying to standardize imports of modules in the same package using relative paths, like:

```Python
from ...callbacks import Callback as _Callback
from ...epics import SiriusPVTimeSerie as _SiriusPVTimeSerie
from ...envars import VACA_PREFIX as _vaca_prefix

from ..csdev import \
    Const as _Const, get_lifetime_database as _get_database
```

what do you think? It is not clear to me whether relative is preferable over absolute paths. I find absolute easier to read but I noticed that trend among us the trend was to use relative paths.

File: siriuspy/siriuspy/opticscorr/chrom/main.py [hunk 2]
```diff
@@ -0,0 +1,551 @@
+"""Main module of AS-AP-ChromCorr IOC."""
+
+import time as _time
+from epics import PV as _PV
+
+from siriuspy import util as _util
```
Comment (xresende) [inline siriuspy/siriuspy/opticscorr/chrom/main.py]: same here as above

File: siriuspy/siriuspy/psdiag/csdev.py [hunk 3]
```diff
@@ -12,15 +12,15 @@ class ETypes(_csdev.ETypes):
         'Current-(SP|Mon) differ', 'Interlocks', 'Wfm error')
 
 
-_et = ETypes  # syntactic sugar
```
Comment (xresende) [inline siriuspy/siriuspy/psdiag/csdev.py]: I had corrected this typo in the previous PR. You are reverting it?

File: siriuspy/siriuspy/psdiag/csdev.py [hunk 4]
```diff
@@ -12,15 +12,15 @@ class ETypes(_csdev.ETypes):
...
+_et = ETypes  # syntatic sugar
 
 
 def get_ps_diag_propty_database(psname):
     """Return property database of diagnostics for power supplies."""
     pstype = _PSSearch.conv_psname_2_pstype(psname)
     splims = _PSSearch.conv_pstype_2_splims(pstype)
     dtol = splims['DTOL_CUR']
-    dbase = {
```
Comment (xresende) [inline siriuspy/siriuspy/psdiag/csdev.py]: What do think of using _dbase_ name in these situations? In the previous PR we replaced _db_ by _dbase_ in various locations in order to comply with pylint three-lettered name rule.

File: siriuspy/siriuspy/currinfo/lifetime/main.py (hunk 1 above)
Comment (anacso17) [reply siriuspy/siriuspy/currinfo/lifetime/main.py]: I also find absolute easier to read, but I will follow the trend. I will commit the changes soon

File: siriuspy/siriuspy/opticscorr/chrom/main.py (hunk 2 above)
Comment (anacso17) [reply siriuspy/siriuspy/opticscorr/chrom/main.py]: see previous comment

File: siriuspy/siriuspy/psdiag/csdev.py (hunk 3 above)
Comment (anacso17) [reply siriuspy/siriuspy/psdiag/csdev.py]: That was a merge error, I will fix that!

File: siriuspy/siriuspy/psdiag/csdev.py (hunk 4 above)
Comment (anacso17) [reply siriuspy/siriuspy/psdiag/csdev.py]: see previous comment

**Pull request closed without merging**